src/components/TournamentMarker.jsx

```diff
@@ -0,0 +1,53 @@
+import {useEffect, useRef, memo, useContext, useCallback} from 'react';
+
+import L from 'leaflet';
+import * as ReactLeaflet from 'react-leaflet';
+
+import icon from "leaflet/dist/images/marker-icon.png";
+import icon2x from "leaflet/dist/images/marker-icon-2x.png";
+import iconShadow from "leaflet/dist/images/marker-shadow.png";
+
+import VisibleTournamentsTrackingContext from '../contexts/VisibleTournamentsTrackingContext';
+
+let defaultIcon = new L.Icon.Default();
+
```

Comment:
Let's carry over the comment from the original location as well.

Reply:
Done.